File: modules/modPing.py
# modPing.py
from modules.mod import mod

class module(mod):
	def __init__(self):
		super().__init__()
		self.description = "Reply `Pong!`"
		self.how = "`?ping`"

	async def run(self, *args):
		del args
		return "Pong!"

File: bot/modules/modPing.go
package modules

var ModPing ModulePing = ModulePing{
	Module{
		Keys: []string{
			"ping",
		},

		Name:        "Ping",
		Description: "Responds with 'Pong!'",
		How:         "`?ping`",
	},
}

type ModulePing struct {
	Module
}

func (mod *ModulePing) Run(args []string) (resp string) {
	return "Pong!"
}

func (mod *ModulePing) Super() Module {
	return mod.Module
}
